apl/scalar-functions/datetime-functions.mdx

```diff
@@ -1,39 +1,40 @@
+| [datetime_diff](#datetime-diff) | Calculates calendarian difference between two datetime values.                                                       |
+| [dayofmonth](#dayofmonth)       | Returns the integer number representing the day number of the given month                                            |
+| [dayofweek](#dayofweek)         | Returns the integer number of days since the preceding Sunday, as a timespan.                                        |
+| [dayofyear](#dayofyear)         | Returns the integer number represents the day number of the given year.                                              |
+| [endofyear](#endofyear)         | Returns the end of the year containing the date                                                                      |
+| [getmonth](#getmonth)           | Get the month number (1-12) from a datetime.                                                                         |
+| [getyear](#getyear)             | Returns the year part of the `datetime` argument.                                                                    |
+| [hourofday](#hourofday)         | Returns the integer number representing the hour number of the given date                                            |
+| [endofday](#endofday)           | Returns the end of the day containing the date                                                                       |
+| [now](#now)                     | Returns the current UTC clock time, optionally offset by a given timespan.                                           |
+| [endofmonth](#endofmonth)       | Returns the end of the month containing the date                                                                     |
+| [endofweek](#endofweek)         | Returns the end of the week containing the date.                                                                     |
+| [monthofyear](#monthofyear)     | Returns the integer number represents the month number of the given year.                                            |
+| [startofday](#startofday)       | Returns the start of the day containing the date                                                                     |
+| [startofmonth](#startofmonth)   | Returns the start of the month containing the date                                                                   |
+| [startofweek](#startofweek)     | Returns the start of the week containing the date                                                                    |
+| [startofyear](#startofyear)     | Returns the start of the year containing the date                                                                    |
+| [unixtime_seconds_todatetime](/apl/scalar-functions/datetime-functions/unixtime-seconds-todatetime)     | Converts a Unix timestamp to an APL `datetime` value.                                                                    |
```

Comment:
Would probably be good to make this a link to: https://axiom.co/docs/apl/data-types/scalar-data-types#the-datetime-data-type